schema.json

```diff
@@ -0,0 +1,514 @@
+      },
+      "EnumMembers": {
+        "$ref": "#/$defs/enumMembers"
+      },
+      "Events": {
+        "$ref": "#/$defs/events"
+      },
+      "DocumentationComments": {
+        "$ref": "#/$defs/documentationComments"
+      }
+    },
+    "additionalProperties": false,
+    "required": [
+      "FullName"
+    ]
+  },
+  "$defs": {
+    "statements": {
+      "type": "array",
+      "items": {
+        "anyOf": [
+          {
+            "$ref": "#/$defs/forEach"
+          },
+          {
+            "$ref": "#/$defs/if"
+          },
+          {
+            "$ref": "#/$defs/switch"
+          },
+          {
+            "$ref": "#/$defs/invocation"
+          },
+          {
+            "$ref": "#/$defs/assignment"
+          },
+          {
+            "$ref": "#/$defs/return"
+          }
+        ]
+      }
+    },
+    "forEach": {
+      "type": "object",
+      "properties": {
+        "$type": {
+          "const": "LivingDocumentation.ForEach, LivingDocumentation.Statements"
+        },
+        "Expression": {
+          "type": "string"
+        },
+        "Statements": {
+          "$ref": "#/$defs/statements"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "Expression"
+      ]
+    },
+    "if": {
+      "type": "object",
+      "properties": {
+        "$type": {
+          "const": "LivingDocumentation.If, LivingDocumentation.Statements"
+        },
+        "Sections": {
+          "type": "array",
+          "items": {
+            "Condition": {
+              "type": "string"
+            },
+            "Statements": {
+              "$ref": "#/$defs/statements"
+            }
+          }
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "Sections"
+      ]
+    },
+    "switch": {
+      "type": "object",
+      "properties": {
+        "$type": {
+          "const": "LivingDocumentation.Switch, LivingDocumentation.Statements"
+        },
+        "Expression": {
+          "type": "string"
+        },
+        "Sections": {
+          "type": "array",
+          "items": {
+            "Labels": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "Statements": {
+              "$ref": "#/$defs/statements"
+            }
+          }
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "Expression"
+      ]
+    },
+    "invocation": {
+      "type": "object",
+      "properties": {
+        "$type": {
+          "const": "LivingDocumentation.InvocationDescription, LivingDocumentation.Descriptions"
+        },
+        "ContainingType": {
+          "type": "string"
+        },
+        "Name": {
+          "type": "string"
+        },
+        "Arguments": {
+          "$ref": "#/$defs/arguments"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "ContainingType",
```

Comment:
I agree that `ContainingType` should be required, but it seems to be missing in exactly one instance in the Pitstop JSON example from [here](https://github.com/eNeRGy164/LivingDocumentation.Workshop/blob/main/2.living-documentation/solutions/21.LivingDocumentation/pitstop.analyzed.json) - do you know what's going on there?

Reply:
Do you have a line number for the example?

Reply:
I came across the issue as well now, same as what I set for Type (for arguments): unless we plan on changing that in the c# implementation we can't set it as required. I removed it for now